### ISSUE TITLE
proposal: add benchmark example in TechEmpower

### ISSUE THREAD

**Issue by Skyenought** [trpc-group/tnet#19]
## Proposal

As a high performance networking library for **go**, is it possible to add benchmark examples for **tnet** like `gnet`? I'm sure this would be **very helpful** for **trpc** promotion.

https://www.techempower.com/benchmarks/#section=test&runid=99e0b150-7cac-4fe7-9b10-5eb526d877ef&hw=ph&test=plaintext

**Comment by WineChord** [comment]
Hi, thank you for your interest in this project. We believe that each networking library is designed to meet specific needs for various business scenarios, with adjustable flags/options to cater to different requirements. In such cases, the results are susceptible to inflation, akin to 'test set leakage' or 'overfitting' in machine learning jargon.

The networking library will continue to evolve and be fine-tuned based on user feedback. Currently, leaderboard ranking is not our top priority.
